### ISSUE TITLE
App crashing randomly

### ISSUE THREAD

**Issue by Deadlander** [KyleADOlson/CombatManager#3]
I have been having issues with the app closing randomly, primarily when I right click something off the Actor menu's (IE rolling an attack, editing, etc).  Is there an error log I can send you from the app?

**Comment by KyleADOlson** [comment]
Yes, there are logs.  Look for C:\users\<username>\appdata\Combat Manager\  You should find some error*.txt files.

**Comment by Deadlander** [comment]
Sorry it took a few days, have been out on vacation... here is the error text file.
[error636365365211597864.txt](https://github.com/KyleADOlson/CombatManager/files/1185239/error636365365211597864.txt)

**Comment by islamelnabarawy** [comment]
+1

I can confirm getting the same issue multiple times.
